RoboJackets/urc-software
# [sensors] publish imu data

# Description
This PR does the following:
- Uses a simple copy of the nmea_navsat_driver code to read IMU data from a USB port.

# Testing Steps (if relevant)
1. `colcon build`
2. `. install/setup.bash`
3. `ros2 launch urc_bringup bringup.launch.py`

Expectation: If you have the IMU USB connected, it should output that data as Quaternions to /imu.

# Self Checklist
- [ ] I have formatted my code using `ament_uncrustify --reformat` (DOES NOT APPLY)
